src/content/docs/codestream/start-here/install-codestream.mdx

```diff
@@ -1,24 +1,55 @@
 ---
-title: Install CodeStream
+title: Install New Relic CodeStream
 metaDescription: "How to install CodeStream and integrate it with your New Relic account."
 ---
 
-CodeStream is a developer collaboration platform that makes it easy to discuss and review code in a more natural and contextual way. 
+New Relic CodeStream is a developer collaboration platform that makes it easy to discuss and review code in a more natural and contextual way. Once connected to New Relic, collaborate on your application errors directly in your IDE.
 
 ## Install CodeStream [#install]
 
-Install CodeStream for [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=CodeStream.codestream), [Visual Studio](https://marketplace.visualstudio.com/items?itemName=CodeStream.codestream-vs) or [JetBrains](https://plugins.jetbrains.com/plugin/12206-codestream).
-
-You can also install the CodeStream extension directly via your Visual Studio Code/Visual Studio's extensions or JetBrains plugins menus.
+You can install CodeStream for your specific IDE or install it through our Instant Observability (I/O) quickstart.
+
+<CollapserGroup>
+  
+    <Collapser
+    className="freq-link"
+    id="io"
+    title="Instant Observability (I/O) quickstart"
```

Comment:
Is this the "preferred" way to install CodeStream? Just want to make sure the ordering of these makes sense.

Reply:
It's what makes the most sense to me and I haven't heard otherwise.